src/utils/DateCalculate.js

```diff
@@ -0,0 +1,67 @@
+export function compareDateDifferences(rawDataTime) {
+  const currentTime = new Date();
+  const time = new Date(rawDataTime);
+
+  const timeDifference = currentTime - time; // 시간 차
+
+  const result = answerDateDifferences(destructureTime(timeDifference));
+  return result;
+}
+
+function destructureTime(timeDifference) {
+  const { minutes: minutesDifference, hours: hoursDifference, days: daysDifference, months: monthsDifference, years: yearsDifference } = {
+    minutes: timeDifference / (1000 * 60),
+    hours: (timeDifference / (1000 * 60)) / 60,
+    days: ((timeDifference / (1000 * 60)) / 60) / 24,
+    months: (((timeDifference / (1000 * 60)) / 60) / 24) / 30,
+    years: ((((timeDifference / (1000 * 60)) / 60) / 24) / 30) / 12,
+  };
+
+  return { minutesDifference, hoursDifference, daysDifference, monthsDifference, yearsDifference };
+}
+
+function answerDateDifferences(allTimeTypes) {
+  const { minutesDifference, hoursDifference, daysDifference, monthsDifference, yearsDifference } = allTimeTypes;
+
+  let result;
+  switch (true) {
+    case minutesDifference < 2:
+      result = "1 minute ago";
+      break;
```

Comment:
임시변수를 두는 것 보다 바로 바로 return을 해줘도 괜찮을 것 같아요~
```suggestion
  switch (true) {
    case minutesDifference < 2:
      return "1 minute ago";
```